webapp/src/features/Dashboard/Dashboard.js

```diff
@@ -1,11 +1,34 @@
-import React from 'react'
+import React, { useEffect } from 'react'
 import SubscriptionTable from './SubscriptionTable'
 import ReportCard from './ReportCard'
 import DashboardFilters from './DashboardFilters'
 import Row from 'react-bootstrap/Row'
 import './Dashboard.css'
+import { useDispatch, useSelector } from 'react-redux'
+import { fetchAllSubscriptions } from './dashboard.asyncActions'
+import {
+	selectSearchType,
+	selectSynchronize,
+	selectSearchFieldValue,
+} from './dashboard.selectors'
 
 const Dashboard = () => {
+	const dispatch = useDispatch()
+	const type = useSelector(selectSearchType)
```

Comment:
use full name
like searchType
because type can be anything..not understood with `type`

Reply:
updated with 'searchType'.